Sources/StreamChatSwiftUI/ChatChannel/MessageList/Polls/PollAttachmentView.swift

```diff
@@ -181,16 +181,23 @@ struct SuggestOptionModifier: ViewModifier {
             content
                 .alert(title, isPresented: $showingAlert) {
                     TextField(L10n.Alert.TextField.pollsNewOption, text: $text)
-                    Button(L10n.Alert.Actions.cancel) {
+                    Button(L10n.Alert.Actions.cancel, role: .cancel) {
                         showingAlert = false
                     }
                     Button(L10n.Alert.Actions.add, action: submit)
                 } message: {
                     Text("")
                 }
         } else {
-            // TODO: Add for iOS < 15.
             content
+                .uiAlertController(
```

Comment:
I was also considering using this for all the iOS versions which makes it easier for us to test this code and for making sure it works the same. WDYT, should we use it for all the iOS versions?

Reply:
we could use one - but first let's fix the bug (see the other comment)